packages/react/src/components/TableCard/tableCardUtils.jsx

```diff
@@ -104,6 +104,7 @@ export const handleExpandedItemLinks = (row, expandedRow, cardVariables) => {
       updatedExpandedRow.push({
         ...item,
         linkTemplate: {
+          ...linkTemplate,
```

Comment:
Previously, without this spread, the target on expandedColumn links was being lost.